README.md

```diff
@@ -55,13 +55,14 @@ effective tool for finding and removing individual records within objects stored
 in S3 buckets. In order to achieve this goal the solution has adopted the
 following design principles:
 
-1. **Security by design:** Every component implements least privilege access,
-   encryption is performed at all layers at rest and in transit, authentication
-   is provided out of the box, expiration of logs is configurable, and record
-   identifiers (known as **Match IDs**) are automatically obfuscated or
-   irreversibly deleted as soon as possible when persisting state.
-2. **Built to scale:** The system has been designed and battle tested for
-   performance and high availability with Data Lakes of TB/PB size.
+1. **Secure by design:** Every component is implemented with least privilege
```

Comment:
this looks duplicated from the Architecture page, is that intended?

Reply:
Yes, that was my goal